### ISSUE TITLE
Anchor Integration and EOS transaction package

### ISSUE THREAD

**Issue by UrsaPolarisRecords** [UrsaPolarisRecords/Polaris#7]
#### User Story
As a _**Polaris user**_, I would like _**to be able to log in with Anchor**_ in order to _**easily send an EOS transaction containing the album liner note data**_.

#### Description
[insert additional feature/bug description, if available]

#### Solution
[add proposed solution, groomed feature description]

#### Design

- [ ] 

#### Smart Contract

- [ ] 

#### Backend

- [ ] 

#### Frontend

- [ ] 

#### Infrastructure

- [ ] 

#### Testing

- [ ] Test in DEV
- [ ] Test in PROD

#### Marketing

- [ ] 

#### Documentation

- [ ]

**Comment by brandonlovejoy** [comment]
Yes, Andrew... Everyone would like to be able to log in with Anchor in order to easily send an EOS transaction. How do I begin?
